packages/component/src/SendBox/TextBox.js

```diff
@@ -147,44 +153,56 @@ const TextBox = ({ className }) => {
 
   return (
     <form
-      className={classNames(ROOT_CSS + '', sendBoxTextAreaStyleSet + '', sendBoxTextBoxStyleSet + '', className + '')}
-      onSubmit={handleSubmit}
+      aria-disabled={disabled}
+      className={classNames(
+        ROOT_CSS + '',
+        sendBoxTextAreaStyleSet + '',
+        sendBoxTextBoxStyleSet + '',
+        'webchat__send-box-text-box',
+        className + ''
+      )}
+      onSubmit={disabled ? PREVENT_DEFAULT_HANDLER : handleSubmit}
```

Comment:
useMemo?

Reply:
Seems overkill for `useMemo` on a ternary operator. 😁